editFamilyProfile.php

```diff
@@ -312,7 +319,7 @@
                     <input type="text" id="econtact-relation" name="econtact-relation" value="<?php echo htmlspecialchars($econtact_relation); ?>">
                 </fieldset>
 
-                <input type="submit" href="familyView.php" name="profile-edit-form" value="Update Profile">
+                <input type="submit" name="profile-edit-form" value="Update Profile">
             </form>
             <a class="button cancel" href="familyAccountDashboard.php" style="margin-top: .5rem">Cancel</a>
```

Comment:
This takes the user to the family account dashboard even if they are logged in as staff.